docker-compose.yml

```diff
@@ -46,6 +45,7 @@ services:
             - db
             - rabbitmq
             - redis
+            - search
```

Comment:
@alastair should we keep this line then?

Reply:
yes, this is still required, see the following lines:

```yaml
    search:
        image: solr:5.5.5
```

the service is still called `search` so we need to keep it in the depends_on for the website